src/cookie.js

```diff
@@ -42,6 +42,12 @@ function setCookie(name, value, options = {path: '/'}) {
     - httpOnly (bool) - if true, cookie won't be available for using in JavaScript
                         IT CAN'T BE FALSE
   */
+  if (!name) {
+    return;
+  }
+
+  options = options || {};
```

Comment:
????

Reply:
If anyone pass null to options code is broken.
So we need to do this casting.

This is basic pattern for working with arguments without type-hinting or strict types